### ISSUE TITLE
Year showing in bibtexparser 0.5.5 but not 0.6

### ISSUE THREAD

**Issue by bamos** [sciunto-org/python-bibtexparser#45]
Hi, in [this bibtex file](https://github.com/bamos/cv/blob/master/publications.bib), bibtexparser 0.5.5 can correctly read the year from `amos2013applying`,
but bibtexparser 0.6.0 doesn't show a year.

I'm using the following as a short example showing this between the 2 versions.
Can you take a look at this when you get a chance?

Regards,
Brandon.

---

```
#!/usr/bin/env python3

from bibtexparser.customization import *
from bibtexparser.bparser import BibTexParser


with open('publications.bib', 'r') as f:
  p = BibTexParser(f.read(), author).get_entry_list()
[print(x) for x in p]
```

# 0.5.5 Output

```
{'booktitle': "IWCMC'13 Security, Trust and Privacy Symposium", 'title': 'Applying machine learning classifiers to dynamic Android\nmalware detection at scale', 'id': 'amos2013applying', 'type': 'inproceedings', 'year': '2013', 'author': ['Amos, Brandon', 'Turner, Hamilton', 'White, Jules']}
```

# 0.6 Output

```
{'type': 'inproceedings', 'author': ['Amos, Brandon', 'Turner, Hamilton', 'White, Jules'], 'id': 'amos2013applying', 'title': 'Applying machine learning classifiers to dynamic Android\nmalware detection at scale', 'booktitle': "IWCMC'13 Security, Trust and Privacy Symposium"}
```

**Comment by sciunto** [comment]
TODO: check is solved by #64

**Comment by sciunto** [comment]
Fixed by #64
